store/seed/test/10000__reset.sql

```diff
@@ -32,6 +32,9 @@ DELETE FROM
 DELETE FROM
     issue;
 
+DELETE FROM
```

Comment:
nit: put before DELETE FROM task.

Reply:
Isn't it already the case?

Reply:
I guess you mean right above `DELETE FROM task`